code/src/java/pcgen/io/PCGIOHandler.java

```diff
@@ -257,7 +257,7 @@ public void read(PlayerCharacter pcToBeRead, InputStream in,
 		}
 	}
 
-	private boolean isPCGCersion2(List<String> lines)
+	private static boolean isPCGCersion2(List<String> lines)
```

Comment:
@jdempsey-au What do you think here?  I don't think we have a standard on this.  Honestly, I'm ambivalent to having things being static or not (don't have a strong opinion), so not sure it matters, but if things are going to be cleaned up, we should decide if this is worth the effort to be changed.